app/src/main/java/org/simple/clinic/scheduleappointment/PatientFacilityChangeActivity.kt

```diff
@@ -0,0 +1,28 @@
+package org.simple.clinic.scheduleappointment
+
+import android.content.Context
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import io.github.inflationx.viewpump.ViewPumpContextWrapper
+import org.simple.clinic.ClinicApp
+import org.simple.clinic.R
+import org.simple.clinic.util.LocaleOverrideContextWrapper
+import java.util.Locale
+import javax.inject.Inject
+
+class PatientFacilityChangeActivity : AppCompatActivity() {
```

Comment:
I am not too sure about the name `PatientFacilityChangeActivity` since this class has nothing to do with changing the patient's facility at the moment. What it does is just lets the user pick a facility and returns the result. Should we just rename it to something like `FacilityPickerActivity` instead?

Reply:
I will rename it to `FacilitySelectionActivity`.